### PR TITLE
Fix for TSA error logging on multi-asic

### PR DESCRIPTION
**Pull request by tjchadaga**
#### Why I did it
TSA/TSB scripts check current state and report error if the state is already in Maintenance/Normal mode. This lookup was not done per DB instance

#### How I did it
Changed state lookup per DB instance

#### How to verify it
Execute TSA twice on multi-asic and check that the error message - "System already in Maintenance" is logged for each DB instance

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [x] 201911
- [ ] 202006
- [x] 202012
- [ ] 202106
- [ ] 202111
- [x] 202205

#### Link to config_db schema for YANG module changes

TSA/B script output on Multi-asic
[log_and_err_message_output.txt](https://github.com/Azure/sonic-buildimage/files/9226405/log_and_err_message_output.txt)

### CODE REVIEW

**Review by judyjoseph — APPROVED** [review]
LGTM, Add the test logs/err messages to the PR description will be helpful

File: dockers/docker-fpm-frr/base_image_files/TS [hunk 1]
```diff
@@ -28,10 +32,16 @@ if [[ ($NUM_ASIC -gt 1) ]]; then
...
             echo -e "BGP"$asic" : \c"
-            if [[ -n "$TSA_STATE_UPDATE" ]]; then 
-                sonic-cfggen -a "$TSA_STATE_UPDATE" -w -n $NAMESPACE_PREFIX$asic
-                logger -t $1 -p user.info "BGP$asic: $log_msg"
-                echo "$log_msg"
+            if [[ -n "$TSA_STATE_UPDATE" ]]; then
+                current_tsa_state="$(sonic-cfggen -d -v BGP_DEVICE_GLOBAL.STATE.tsa_enabled -n $NAMESPACE_PREFIX$asic)"
+                if [[ $current_tsa_state  == $desired_tsa_state ]]; then
+                    echo "$err_msg"
+                    logger -t TSA -p user.info "$err_msg"
```
Comment (arlakshm) [inline dockers/docker-fpm-frr/base_image_files/TS:39]: `TSA` should be $1?

Comment (tjchadaga) [reply dockers/docker-fpm-frr/base_image_files/TS]: thanks for catching this! fixed the issue

**Comment by tjchadaga** [comment]
> LGTM, Add the test logs/err messages to the PR description will be helpful

Updated

**Comment by qiluo-msft** [comment]
This commit could not be cleanly cherry-picked to 202012. Please submit another PR.

**Pull request merged**